vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -262,21 +264,23 @@ def _load_model_weight_or_group_weight_scale(self, shard_dim: int,
                                                  expert_data: torch.Tensor,
                                                  shard_id: str,
                                                  loaded_weight: torch.tensor,
-                                                 tp_rank: int):
+                                                 tp_rank: int, expert_id: int):
```

Comment:
load_w2/13 are called from multiple places, did not want to change the signature of all the methods that are calling it, therefore I made the argument optional, regarding the calling method that is called on hpu during INC quantization and regular flow (no quant) I've added the param as required since it is used by it.
regarding the format of the arguments, this is the way the vllm formatter aligns the args, if I change it to a separate line it will fail format verification.